plotting.py

```diff
@@ -112,13 +124,20 @@ def plot_histogram(_team_alerts, _team_labels, experiment_name, suricata_summary
     plt.tick_params(axis='x', which='major', labelsize=8)
     plt.tick_params(axis='x', which='minor', labelsize=8)
     # plt.yticks(np.arange(0, 13000, 1000))
-    plt.legend([plot[0] for plot in plots], _team_labels)
+    plt.legend([plot[0] for plot in plots], team_labels)
     plt.tight_layout()
     plt.savefig('data_histogram-' + experiment_name + '.png')
     # plt.show()
 
 
 def plot_episodes(frequencies, episodes, mcat):
+    """
+    Plot the slopes based on the time, for a given (hyper)alert sequence (for an attacker-victim pair and mcat).
```

Comment:
Is this more or less the purpose of this method?

Reply:
Plot the rate of alerts over time, for a given hyperalert sequence of specific mcat and belonging to an attacker-victim pair

Reply:
Updated